deployment/deployment.json

```diff
@@ -7965,6 +7965,102 @@
       }
     }
   },
+  "orbit": {
```

Comment:
can utilize the already existing `orbit-bridge` configuration in this file

Reply:
I've just deleted that configuration given it wasn't being used.